docs/markdown/snippets/shared_static_only_args.md

```diff
@@ -0,0 +1,9 @@
+## `<lang>_(shared|static)_args` for both_library, library, and build_target
+
+We now allow passing arguments like `c_static_args` and `c_shared_args`. This
+allows a `both_library` to have arguments specific to either the shared or
```

Comment:
s/both_library/both_libraries/g

(We could use the [[]] form to generate inter-documentation links.)